src/components/NewsletterForm/_newsletter-form.scss

```diff
@@ -39,7 +39,7 @@
 }
 
 .newsletter-form__item--footer {
-  font-size: var(--text-xs);
+  font-size: 12px;
```

Comment:
should this be a variable?

Reply:
ideally - though we don't have a 12px font-size as a typography variable (see `_typography.scss`...

I tried this using the `--text-xs` variable, but it looks a lot different to the current Footer (and we need to be consistent with the footer styles as we'll be serving one from D7 and this new one!)